### DISCUSSION TITLE
Iota lotterie

### DISCUSSION THREAD

**Discussion by yearning69** [iotaledger/firefly#8192] [Help & Support]
Hi, Ich wurde gerade von meinem Wallet auf eine Seite verlinkt . Mit der Aussicht auf eine Iota -lotterie. Ich habe mein Backup eingegeben und ich glaube mein Wallet ist jetzt leer. 4000Iota weg. Kann man da noch was machen?

**Comment by no8ody** [comment]
Hi,

leider kann man da nichts mehr machen, wenn du deinen Seed durch deinen Backup an die Betrüger übertragen hast und diese dein Guthaben transferiert haben. Daher wird auch eine entsprechende Warnung bzgl. der Lotterie oder anderen Links in Firefly angezeigt.

Das einzige was du machen könntest, ist eine entsprechende Anzeige bei deiner lokalen Polizeidienststelle aufzugeben und zu hoffen, dass der/die Betrüger gefasst werden und die Funds sichergestellt werden können.

Du solltest niemals auf irgendwelchen Webseiten oder sonstigen externen Quellen deinen Seed eingeben, daher beim nächsten Mal lieber einmal länger darüber nachdenken und zur Not versuchen sich vorher zu erkundigen.

**Reply by yearning69** [reply]
Ja danke für die schnelle Antwort. Ich hätte nicht gedacht das externe betrügerische Nachrichten in mein Wallet erscheinen. Mist.

**Reply by no8ody** [reply]
Der Tangle ist öffentlich, wie bei so ziemlich jedem DLT und somit kann auch jeder die Adresse mit Guthaben rausfinden und entsprechend z.B. ein NFT an diese Adresse schicken. Daher ist es eben umso wichtiger sein Seed so gut wie möglich zu schützen, da du in diesem Fall deine eigene Bank bist.

**Reply by PG79Linz** [reply]
Mir ist das auch pasiert, ich finde es PERVERS das dies auf der offiziellen Wallet von IOTA so einfach geht und die Nichts unternehmen.
Wenn ich sowas lese vertraue ich drauf dass es von iota ist!!!
SCHWEINE!

**Comment by yearning69** [comment]
Ok, schmerzlich, schlauer durch aua.

**Comment by georgeparker88** [comment]
holy shit scammers can send messages to your wallet?

**Comment by georgeparker88** [comment]
fuck just lost a lot of money being stupid

**Comment by georgeparker88** [comment]
Anybody have an idea how to see where the funds have gone?

**Comment by tackar66** [comment]
das ist für mich auch bitter, 
habe auch eine Menge Iota verloren dadurch, würde mich auch Interessieren wohin alle Token gekommen sind, da es ja noch nicht einmal einen Transaktionsverlauf darüber gibt.

**Reply by begonaalvarezd** [reply]
you can use the explorer

**Comment by begonaalvarezd** [comment]
I am very sorry you are a victim of the iota lottery scam. We cant prevent people from gifting you an NFT, addresses are public so anybody can pick an address and make a gift, unfortunately in this case is an NFT with a scam message. The whole point of self custody wallets in crypto is that you are you own bank so you have to take good care of your keys because you are the only one holding them. The ecosystem is full of warnings like "do not share your keys with anybody, do not enter your keys in any random apps, ..." The thief didnt go to your wallet, but you went to a random website and gave your keys. At this point we can only suggest you to report it to your local authorities.

**Reply by abdullahcolak1** [reply]
Hello, today I lost all IOTA due to lottery fraud. You wrote that nothing can be done. But you talk about how secure the Firefly app is.  
" Firefly guides you through the complexities of crypto, protected and with ease while using cutting-edge technology to secure your tokens. Written in Rust: Rust offers security from the ground up – while maintaining speed, low memory usage, and scalability. This allows us to build apps with better performance and higher security.”
With one click you lose all your Iota. I don't understand what kind of safe application this is. I started using this application because it was safe, but my 3 years of effort were wasted. Why don't you mention this kind of security vulnerability when explaining the application?

**Comment by andreuenigma** [comment]
Mis IOTAs y mis Shimmer han desaparecido con lo de IOTA Lottery! ¡¿Cómo es posible un scam dentro de la misma aplicación Firefly?!
